schema/proto3/subscribeinfoprovisioner.proto

```diff
@@ -0,0 +1,43 @@
+/*
+MIT License
+Copyright(c) 2020 Futurewei Cloud
+
+    Permission is hereby granted,
+    free of charge, to any person obtaining a copy of this software and associated documentation files(the "Software"), to deal in the Software without restriction,
+    including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and / or sell copies of the Software, and to permit persons
+    to whom the Software is furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+    WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+
+syntax = "proto3";
+
+package alcor.schema;
+
+option java_package = "com.futurewei.alcor.schema";
+
+service SubscribeInfoProvisioner {
+
+  rpc PushNodeSubscribeInfo (NodeSubscribeInfo) returns (SubscribeOperationReply) {
+  }
+
+}
+
+enum SubscribeOperation {
+  Subscribe = 0;
+  Unsubscribe = 1;
+}
+
+message NodeSubscribeInfo {
+  SubscribeOperation subscribe_operation = 1;
+  string topic = 2;
+  string key = 3;
+}
+
+message SubscribeOperationReply {
+  bool is_success = 1;
```

Comment:
For the response, we can use an existing operation response entity: 

https://github.com/futurewei-cloud/alcor/blob/64f5d8315c9c7d1335c4593438e09285f725efce/schema/proto3/common.proto#L43

Reply:
I will use OperationStatus to replace the bool variable.